www/docs/style/sass/app.scss

```diff
@@ -162,12 +162,16 @@ a {
     color: $links;
 }
 
-a:hover,
-a:focus {
+a:hover {
     text-decoration: underline;
     color: darken($colour_primary, 10%);
 }
 
+a:focus {
```

Comment:
The change to focus is nothing to do with this commit, should be its own/different commit.